variables.tf

```diff
@@ -128,6 +128,12 @@ variable "enable_azure_policy" {
   default     = false
 }
 
+variable "enable_open_service_mesh" {
+  description = "Enable Open Service Mesh Addon."
```

Comment:
Can we use the same description as [provider's document](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/website/docs/r/kubernetes_cluster.html.markdown?plain=1#L152)?